### ISSUE TITLE
Makefile for clang/bcc seem to be "dirty"

### ISSUE THREAD

**Issue by GitMensch** [LADSoft/OrangeC#143]
It contains the bcc entry

```makefile
vpath %.o $(_OUTPUTDIR)
vpath %$(LIB_EXT) c:\bcc55\lib c:\bcc55\lib\psdk $(_LIBDIR)
vpath %.res $(_OUTPUTDIR)
```

Note: I have neither tested bcc nor clang, but I assume at least for clang the bcc-entry is wrong. For bcc the bcc-path could be the standard value but the user should be able to override it (using a variable that may be changed when passed to make, instead of the fixed path)

**Comment by GitMensch** [comment]
Clang part fixed with ec9df9139902b4a326b3e7e54039e7e83b6cd283.

This brings up the additional question to if it is useful to add a clang environment to AppVeyor...

**Comment by LADSoft** [comment]
the bcc32 should have been fixed as well.   That was done with dde65e8682dc3ec9a0a2ded9284b119463a7ddf2 which also fixed various problems the recent code base had with BCC32 compatibility. Sorry i didn't attribute those changes properly.

As far as compiling with clang we can leave this open and I'll look into it...   but as part of this I did try compiling with clang and it seems the current 64-bit version at least may have a bug.   In any case I wasn't able to complete the compile due to errors it threw...

Maybe it would work better if I loaded the 32-bit version of clang though.
